Sources/MongoSwift/BSON/Overwritable.swift

```diff
@@ -43,9 +43,11 @@ extension Double: Overwritable {
 
 extension Decimal128: Overwritable {
     internal func writeToCurrentPosition(of iter: DocumentIterator) throws {
-        // TODO: avoid this copy via withUnsafePointer once swift 4.1 support is dropped (SWIFT-284)
-        var copy = self.decimal128
-        bson_iter_overwrite_decimal128(&iter.iter, &copy)
+        withUnsafePointer(to: self.decimal128) { ptr in
+            // bson_iter_overwrite_decimal128 takes in a (non-const) *decimal_128_t, so we need to pass in a mutable
+            // pointer. no mutation of self.decimal128 should occur, however.
```

Comment:
I saw you opened a CDRIVER ticket, can you link it here please?

Reply:
done